### PR TITLE
Update to L1T Global for 2016 Menu (80X)

### PR DESCRIPTION
**Pull request by mulhearn**
This is the 80x version of #14282.

### CODE REVIEW

**Comment by mulhearn** [comment]
please test

**Comment by mulhearn** [comment]
+1

**Comment by mulhearn** [comment]
This new includes also the IB RelVal bugfix #14292

**Comment by mulhearn** [comment]
please test

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py [hunk 1]
```diff
@@ -24,6 +24,20 @@ def esproducers_by_type(process, *types):
...
 #     return process
 
+def customiseFor14282(process):
+    process.GlobalParametersRcdSource = cms.ESSource("EmptyESSource",
+        recordName = cms.string('L1TGlobalParametersRcd'),
+        iovIsRunNotTime = cms.bool(True),
+        firstValid = cms.vuint32(1))
+    return process
+
+# Add quadruplet-specific pixel track duplicate cleaning mode (PR #13753)
```
Comment (fwyzard) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py]: I don't think we want this one in 8.0.X - though it should be harmless if it not used.

Comment (Martin-Grunewald) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py]: Is fine for now: the ConfDB menus contain the ES producer, so online where this file is not used
is fine. We can clean this up later in the release with new dumps - depending what goes in
first, this should be kept for now.

**Comment by Martin-Grunewald** [comment]
+1

**Comment by mulhearn** [comment]
+1

**Comment by mulhearn** [comment]
(I'll make additional PR to handle the IB failures discovered in 81X)

**Comment by mulhearn** [comment]
Actually, I think safest to just update this PR...  this now includes the bugfixes targetting IB failures in 81x branch:   cms-sw#14292, cms-sw#14296, cms-sw#14297.

**Comment by mulhearn** [comment]
please test

**Comment by mulhearn** [comment]
+1

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py (hunk 1 above)
Comment (fwyzard) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py]: I meant the quadruplet pixel cleaning...

**Pull request merged**